nbs/docs/Prediction_intervals_in_forecasting_models.ipynb

```diff
@@ -0,0 +1,1394 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def snake_case(series , column_name):
</pre><p>snake_case is the term used to describe the style that uses all lowercase and separates words with underscores. What I meant here was that you should name the function augmented_dickey_fuller_test instead of Augmented_Dickey_Fuller_Test, not literally naming it snake_case</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/mlforecast/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>